pyvista/core/filters/uniform_grid.py

```diff
@@ -165,6 +165,89 @@ def extract_subset(self, voi, rate=(1, 1, 1), boundary=False, progress_bar=False
         fixed.copy_meta_from(result)
         return fixed
 
+    def image_dilate_erode(
+        self,
+        dilate_value=1,
+        erode_value=0,
+        kernel_size=(3, 3, 3),
+        scalars=None,
+        preference='points',
+        progress_bar=False,
+    ):
+        """Dilates one value and erodes another.
+
+        ``image_dilate_erode`` will dilate one value and erode another. It uses
+        an elliptical footprint, and only erodes/dilates on the boundary of the
+        two values. The filter is restricted to the X, Y, and Z axes for now.
+        It can degenerate to a 2 or 1-dimensional filter by setting the kernel
+        size to 1 for a specific axis.
+
+        Parameters
+        ----------
+        dilate_value : int or float, optional
+            Dilate value in the dataset. Default: ``1``.
+
+        erode_value : int or float, optional
+            Erode value in the dataset. Default: ``0``.
+
+        kernel_size : list(int) or tuple(int), optional
+            Length 3 iterable of ints: ``(xsize, ysize, zsize)``.
+            Determines the size (and center) of the kernel.
+            Default: ``(3, 3, 3)``.
+
+        scalars : str, optional
+            Name of scalars to process. Defaults to currently active scalars.
+
+        preference : str, optional
+            When scalars are specified, this is the preferred array
+            type to search for in the dataset.  Must be either
+            ``'point'`` or ``'cell'``.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress. Default ``False``.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            Dataset with the specified scalars thresholded.
```

Comment:
Is this "thresholded" a typo from copy-pasting? Seems off.

Reply:
[Thresholded](https://en.wiktionary.org/wiki/thresholded) is a word but I can change it if you think it's weird/awkward.

Reply:
Sorry, I phrased that ambiguously. I meant that the filter doesn't threshold, it erodes/dilates, right? If instead you mean that scalars get transformed in nontrivial ways this should be explained somewhere in the docstring.

Reply:
Oh you are right, this is a copy-paste error. I will fix it in the next commit. Thanks!